delira/training/base_trainer.py

```diff
@@ -59,6 +54,9 @@ def __init__(self,
                  metric_keys=None,
                  convert_batch_to_npy_fn=lambda x: x,
                  val_freq=1,
+                 logging_callback_cls=DefaultLoggingCallback,
```

Comment:
Eventually reorder arguments, such that all logging related arguments are together.

Reply:
I would like this, but this would introduce some breaking changes, which is why I haven't done this. One the other hand: We have lot's of breaking changes, so we might consider doing this now and therefore avoid breaking changes in the future